Modules/CodeTrek/Services/CodeTrekRoundDetailService.php

```diff
@@ -33,6 +35,8 @@ public function takeActionApplicantToRound($applicant)
         $applicationRound->round_name = $applicant->round_name;
         $applicationRound->feedback = null;
         $applicationRound->start_date = today();
+        $codetrekApplicant = CodeTrekApplicant::find($applicant->id);
+        Mail::send(new CodeTrekApplicantRoundMail($applicationRound, $codetrekApplicant));
```

Comment:
Could you please comment on this line for now? For now, we don't need this mail functionality. This mail functionality is enabled when the use of the codeTrek module starts in the proper way.

Also could you please add this line before this line?
`This line will be uncommented in the future when the use of the codeTrek module starts in the proper way.`
```suggestion
        // Mail::send(new CodeTrekApplicantRoundMail($applicationRound, $codetrekApplicant));
```